example-app/src/main/java/com/adyen/checkout/example/ui/googlepay/compose/SessionsGooglePayEvents.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 13/12/2023.
+ */
+
+package com.adyen.checkout.example.ui.googlepay.compose
+
+import android.content.Intent
+import com.adyen.checkout.components.core.action.Action
+
+internal abstract class SessionsGooglePayEvents internal constructor() {
+    object None : SessionsGooglePayEvents()
```

Comment:
Neat! We probably can't have simultaneous events in this flow so this should work